test/WebJobs.Script.Tests.Integration/WebHostEndToEnd/HostConfigurationExceptionTests.cs

```diff
@@ -75,5 +80,27 @@ public void Dispose()
                 Console.WriteLine(ex.Message);
             }
         }
+
+        private class TemporaryTestFunctionHost : TestFunctionHost
+        {
+            public TemporaryTestFunctionHost(string scriptPath, string logPath,
+                Action<IServiceCollection> configureWebHostServices = null,
+                Action<IWebJobsBuilder> configureScriptHostWebJobsBuilder = null,
+                Action<IConfigurationBuilder> configureScriptHostAppConfiguration = null,
+                Action<ILoggingBuilder> configureScriptHostLogging = null,
+                Action<IServiceCollection> configureScriptHostServices = null)
+                : base(scriptPath, logPath, configureWebHostServices, configureScriptHostWebJobsBuilder, configureScriptHostAppConfiguration, configureScriptHostLogging, configureScriptHostServices)
+            {
+            }
+
+            // Overriding this function as currently there's that causes HostState to be "Initialized" in case host.json has errors
+            // TODO: Once the issue is fixed, this class needs to be removed.
+            // https://github.com/Azure/azure-functions-host/issues/5959
```

Comment:
>https://github.com/Azure/azure-functions-host/issues/5959 [](start = 15, length = 57)

Move this to a sprint instead of leaving it at triaged?

Reply:
I am not sure if this is a P1 or P2, so i didn't pull it in a Sprint. It's been like this and the only real issue is that the HostStatus is different than what it should be.

If it becomes bigger issue, maybe worth pulling it in then?